REPOSITORY.md

```diff
@@ -0,0 +1,39 @@
+# Open Cluster Management Repositories Guidelines
+
+This document attempts to outline a structure for creating GitHub repositories with the Open-Cluster-Management project. It also describes how and when repositories are removed.
+
+# Rules for New Repositories
+
+* All repos will live in github.com/open-cluster-management-io/\<project-name\>.
+* Must contain the topic for the sponsoring sub projects - e.g. application. (Added through the Manage topics link on the repo page.)
```

Comment:
which subproject do you think will the proxy repo belong to?

Reply:
I have another PR trying to define the existing sub project https://github.com/open-cluster-management-io/community/pull/75. I am thinking whether it should belong to cluster lifecycle sub project, or we should have a new subproject as `Integrations` to include all project that is to integrate with external projects?